e2e/aks_model.go

```diff
@@ -147,7 +162,6 @@ func getRequiredSecurityRules(clusterFQDN string) ([]*armnetwork.SecurityRule, e
 	requiredDNSNames := []string{
 		"mcr.microsoft.com",
```

Comment:
i think we can try to remove `mcr.microsoft.com` and `management.azure.com`
or maybe next pr?
Btw, i may miss, how the acr is created? is it in another pr?

Reply:
I guess there is no private dns zone and private endpoint in the cluster vnet to connect the acr. Thus we may need to whitelist the acr fqdn as the workaround in this pr?

Reply:
There is a private endpoint for the acr. From what I can tell in the Microsoft docs, we should just need to whitelist the AzureContainerRegistry - I’ve tested these nsg settings in a VM. Why are you thinking we need to whitelist the acr fqdn? Just trying to get more specific on the address that’s whitelisted?

Reply:
the pe for the acr is under vnet named `oras-login-testing-vnet`. Thus, in my understanding, only vm under the same vnet can connect to this acr via private endpoint. Otherwise, it goes through public network. i guess the e2e cluster is not under the same vnet, which causes the e2e failure when acr fqdn `aksvhdtestcr.azurecr.io` is not whitelisted

Reply:
I’m not sure that it is the case, I’ll follow up. From what I can tell, we are hitting a redundant e2e failure not related to the pe. We would be seeing timeouts related to Oras pull if it didn’t have the correct permissions for the acr

Reply:
You're right, so I'm adding some code to create a PE between the airgap e2e resource group and the ACR, that way the allowAllVnet in the nsg should work without a problem :)